src/systemInfo.tsx

```diff
@@ -9,22 +9,15 @@
  */
 
 import React from "react";
-import styles from "./hello.css";
-import { Greeter } from "./greeter/greeter";
-import { PatientGetter } from "./patient-getter/patient-getter";
-import { Boxes } from "./boxes/slot/boxes";
+import styles from "./systemInfo.css";
+import { Tables } from "./tables/slot/tables";
 
-const Hello: React.FC = () => {
+const SystemInformation: React.FC = () => {
```

Comment:
This is something I have a hard time getting used to for OpenMRS project. For me I would name this file `SystemInfo.tsx` (with capital S) for a file whose main export is a React Component. However, unfortunately it is the convention of the community to have the file be kebab case like `system-info.tsx` or `system-info.component.tsx`. I really dislike this system, but that is how the community is structuring their file systems.

Reply:
🤔 I think different companies have different conventions, actually, at my company, we emphasize camelCase for component naming and even functions, I will use kebab case for this.